win32/src/file_win32.c

```diff
@@ -1,31 +1,17 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#include "windows.h"
-#include "azure_c_pal/file.h"
-
-typedef struct FILE_WRITE_DATA_CONTEXT_TAG
-{
-    FILE_WRITE_CB user_callback;
-    void* user_context;
-    const unsigned char* source;
-    uint32_t size;
-}FILE_WRITE_DATA_CONTEXT;
+#include <inttypes.h>
+#include <stdbool.h>
+#include <stdint.h>

```

Comment:
>stdint [](start = 10, length = 6)

not needed if you include inttypes

Reply:
Done

---
In reply to: [456703395](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456703395) [](ancestors = 456703395)